findbugs.properties

```diff
@@ -1,13 +1,19 @@
 category=External Analysers
 description=Provide Findbugs rules for analysis of Java projects
-archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0,3.11.0,3.9.4,3.11.1
-publicVersions=4.0.0
+archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0,3.11.0,3.9.4,3.11.1,4.0.0
+publicVersions=4.0.1
 
 defaults.mavenGroupId=org.sonarsource.sonar-findbugs-plugin
 defaults.mavenArtifactId=sonar-findbugs-plugin
 
+4.0.1.description=Use SpotBugs 4.1.2, sb-contrib 7.4.7, and findsecbugs 1.10.1
+4.0.1.sqVersions=[7.9,LATEST]
+4.0.1.date=2020-09-18
+4.0.1.changelogUrl=https://github.com/spotbugs/sonar-findbugs/releases/tag/4.0.1
+4.0.1.downloadUrl=https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.0.1/sonar-findbugs-plugin-4.0.1.jar
+
 4.0.0.description=Use SpotBugs 4.0.0, sb-contrib 7.4.7, and findsecbugs 1.10.1
-4.0.0.sqVersions=[7.9,LATEST]
+4.0.0.sqVersions=[7.9,8.3]
```

Comment:
This range needs to end with the current version. Otherwise your users on 8.4.* won't be prompted to upgrade in the Marketplace.

Reply:
This version has compatibility problem with latest version so I wanna exclude it. Is it impossible?

Reply:
Not impossible. And okay to do if you don't think anyone will be using the combination. But if you do have someone on SQ8.4 & FB 4.0.0, then you _want_ the Marketplace to prompt an upgrade. Because right now Marketplace is offering 4.0.0 as an install to folks on 8.4.
Also, don't worry that setting the end range to 8.4.* means that folks will be offered 4.0.0 after 4.0.1 is released. Only the latest compatible version of a plugin is ever offered via the Marketplace.